lineman/app/components/memberships_page/cancel_invitation_form/cancel_invitation_form.haml

```diff
@@ -0,0 +1,12 @@
+%form{ng-submit: 'submit()'}
+  .modal-header
+    %button{type: 'button', class: 'close', ng-click: '$close()'}
+      %span{aria-hidden: true} &times;
+      %span.sr-only{translate: 'common.action.cancel'}
+    %h1.lmo-h1{translate: 'cancel_invitation_form.heading'}
+
+  .modal-body
+    %span{translate: 'cancel_invitation_form.question'}
+  .modal-footer
+    %button.btn.btn-danger{type: 'submit', translate: 'cancel_invitation_form.submit'}
```

Comment:
This is what's causing the tests to fail. When I put a BEM class on this button, and then inspect it in the browser, it's not there, so when protractor tries to find the element by css it can't... @robguthrie, suggestions? Also the whole thing might need a quick going over...